lib/actions/device.coffee

```diff
@@ -364,7 +357,7 @@ exports.init =
 				removeCallback(callback)
 
 			(callback) ->
-				resin.models.device.getByUUID(params.uuid, callback)
+				resin.models.device.getByUUID(params.uuid).nodeify(callback)
```

Comment:
This function doesn't exist in the SDK anymore. Since `resin.models.device.get()` handles a uuid now, you can use that function instead.